README.md

```diff
@@ -303,26 +303,45 @@ This project uses a [Zenodo](https://zenodo.org) generated DOI. The badge at the
 
 Many ISIS applications require ancillary data. For example, calibration applications require flat files to do flat field corrections, and map projection applications require DTMs to accurately compute intersections. Due to its size, this data is stored in a separate directory called the ISIS Data Area. Any location can be used for the ISIS Data Area, the software simply requires that the ISISDATA environment variable is set to its location.
 
-### Structure of the ISIS3 Data Area
+### Structure of the ISIS Data Area
 
 Under the root directory of the ISIS Data Area pointed to by the ISISDATA environment variable are a variety of sub-directories. Each mission supported by ISIS has a sub-directory that contains mission specific processing data such as flat files and mission specific SPICE. There are also data areas used by more generic applications. These sub-directories contain everything from templates to test data.
 
-### Size of the ISIS3 Data Area
+### Versions of the ISIS Data Area
+
+In ISIS version 4.1.0, several files previously stored in the data area which were closely associated with ISIS applications were moved into version control with the ISIS source code. Additionally, the environment variables used for the ISIS Data Area, and the rsync location for the ISIS Data Area were also updated. 
+
```

Comment:
Extra comma after "the environment variables used for the Data Area"

Reply:
stylistic choice I'm pretty sure, comma and no comma are correct (I thought the same then looked it up and it seems fine?), if we were to get picky, I would say remove "which were" in the first sentence since it's unnecessary.

Reply:
Appreciate the feedback!